api/api/configuration.py

```diff
@@ -257,6 +262,10 @@ def replace_bools(conf):
     # Append wazuh_path to all paths in configuration
     append_wazuh_path(configuration, [('logs', 'path'), ('https', 'key'), ('https', 'cert'), ('https', 'ca')])
 
+    # Assign API request timeout
+    if config_file == api_constants.CONFIG_FILE_PATH:
+        api_constants.API_REQUEST_TIMEOUT = configuration['intervals']['request_timeout']
```

Comment:
I wonder if we could just use the `api_conf` object as we do in other places to get the value of `request_timeout`, for instance in:
https://github.com/wazuh/wazuh/blob/93d847aa8a8b5f4de4127d9c6179b5d59f921bfc/api/api/controllers/experimental_controller.py#L25
or in:
https://github.com/wazuh/wazuh/blob/93d847aa8a8b5f4de4127d9c6179b5d59f921bfc/framework/wazuh/core/manager.py#L192
instead of using/updating a constant. Is there any reason to do this?